wagn/lib/wagn/application.rb

```diff
@@ -84,14 +84,14 @@ def paths
         
         paths.add 'files'
         
+        #more consistent to call this deck mod...
         paths.add 'local-mod', :with=>'mod'
         paths['app/models'] = []
         paths['app/mailers'] = []
         
-    
-        # should we have add_deck_paths for these?
-        add_gem_path paths, "lib/tasks",           :with => "lib/wagn/tasks", :glob => "**/*.rake"
+        add_gem_path paths, "lib/tasks",     :with => "lib/wagn/tasks", :glob => "**/*.rake"
         add_gem_path paths, 'gem-assets'
+        add_gem_path paths, 'config/routes', :with => 'lib/wagn/config/routes.rb'
```

Comment:
I guess I'm ok with that.  As I think about it, I'm not sure which is the recommended rails practice anyway.  This is for supporting a rails app with its routes in a gem, so I think best rails practice is relevant, but as I said, I'm not sure which one it is.  Seems like putting *un-namespaced* stuff down under lib/wagn or lib/decko would be at least as confusing as putting them in config.  I had a really hard time finding this when I was trying to change where it found it in the initial splitting.